hypervisor/src/mshv/mod.rs

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: Apache-2.0 OR BSD-3-Clause
+//
+// Copyright © 2020, Microsoft Corporation
+//
+
+#![allow(dead_code)]
+#![allow(unused_imports)]
+#![allow(unused_variables)]
+#![allow(unused_macros)]
+#![allow(non_upper_case_globals)]
```

Comment:
These were added to help focus on development. At the time I implemented more things than necessary because I wanted them to be available whenever I needed them.

At least the first four should be dropped when it comes to upstreaming and we should fix whatever warnings that show up. I don't think there are that many.

Reply:
I was thinking to do it in a future PR if you do agree.

Reply:
Fine by me.